### DISCUSSION TITLE
Anchors inside a tabs box?

### DISCUSSION THREAD

**Discussion by Mrt134** [thegeeklab/hugo-geekdoc#353] [Q&A] [answered]
Love the theme, thanks for the effort you put into it!

Is it possible to enable linking to a header that is inside a tabs box so that clicking a link sends the viewer directly to the box with the correct tab selected?

**Comment by xoxys** [comment, marked as answer]
Thanks for your kind words. Regarding your question, that's not possible, sorry.
